client/views/events/edit/event.edit.js

```diff
@@ -263,9 +267,10 @@ Template.eventEdit.events({
 
 		var updateReplicas = instance.$("input[name='updateReplicas']").is(':checked');
 		var sendNotification = instance.$(".js-check-notify").is(':checked');
+		var addNotificationMessage = instance.$("#eventEditAddMessage").val();
```

Comment:
Please use class `js-event-edit-add-message` instead of an ID.

The reason for using classes instead of ID is that having multiple elements with the same ID on one page is illegal. While in a case like this we don't expect having multiple `eventEditAddMessage` fields displayed at the same time, I prefer not having to spend effort and think whether the case could arise.